mmdet/core/anchor/point_generator.py

```diff
@@ -35,3 +37,198 @@ def valid_flags(self, featmap_size, valid_size, device='cuda'):
         valid_xx, valid_yy = self._meshgrid(valid_x, valid_y)
         valid = valid_xx & valid_yy
         return valid
+
+
+@PRIOR_GENERATORS.register_module()
+class MlvlPointGenerator:
+
```

Comment:
This class also need a docstring, also explain what does mlvl means

Reply:
Actually, it should be `PointGenerator`, but it has been used by 
https://github.com/open-mmlab/mmdetection/blob/0e09f00a5bf76169386cadbf4c41bd0ff7f6208d/mmdet/core/anchor/point_generator.py#L7
So I add the prefix `Mlvl` which means `multi-level`, I would explain it in docs.